data/tokens.json

```diff
@@ -211,109 +211,197 @@
       }
     },
     "typography": {
-      "H1": {
+      "family": {
```

Comment:
Organized the font family, font size, font weight, line height, etc. values to live under `typography` instead since they shouldn't be used for any other categories